uftrace.c

```diff
@@ -858,7 +861,17 @@ static int parse_option(struct uftrace_opts *opts, int key, char *arg)
 		break;
 
 	case OPT_disabled:
-		opts->disabled = true;
+		pr_use("'--disable' is deprecated, use --trace=on|off instead.\n");
```

Comment:
I think the message is better to be `use --trace=off instead` because `--trace=on` is not related to `--disable`.

Reply:
Ok. Should I remove `--disable` from the documentation? Or mark it as deprecated there too?

Reply:
Mark as deprecated, please.